src/masternodeman.cpp

```diff
@@ -526,11 +526,10 @@ void CMasternodeMan::ProcessMasternodeConnections()
 
     LOCK(cs_vNodes);
     BOOST_FOREACH(CNode* pnode, vNodes) {
-        if(pnode->fDarkSendMaster){
+        if(pnode->fDarkSendMaster) {
```

Comment:
Good point 👍 Actually shouldn't mention any specifics at all and usage was not described correctly imo (was "where" to use rather then "when"/"why") - fixed var names and comments.